### ISSUE TITLE
Error with commission

### ISSUE THREAD

**Issue by aldeni** [pmorissette/bt#247]
Hi guys, Im new with lib and would like to know whats going on with my backtest.
My backtest is working properly without commission but when a set the commission the code returns to me the following error message:

"Exception: The difference between what we have raised with q and the amount we are trying to raise has gotten bigger since last iteration! full_outlay should always be approaching amount! There may be a case where the commission fn is not smooth"

I set the commission as a percentage of the traded value (eg. 0.15% trade value => q * p * 0.0015).

Im not sure but I think the problem is related to the commission amount could be large...

Tks for the great job with this lib! Its is very simple but powerfull :-)

**Comment by blenderben2** [comment]
I set `comm = lambda q, p: q * p * 0.0015` and `bt.Backtest(s, data, commissions=comm)`. Works perfectly.

**Comment by aldeni** [comment]
Hi, thanks for the assistance. I had solved this some weeks ago but I'm not sure exactly what's happened.

The issue is probably related to the fact that in some rebalances looks like there's no cash available to complete the buy orders or something like that.

To solve this I just increased the start cash amount and then applied the commision formula as you stated.

Also, I searched some way to bypass this cash constrain but dont find any setting regarding this something to disable the cash management
